src/FastSerialization/FastSerialization.cs

```diff
@@ -1979,49 +2039,31 @@ private IFastSerializable ReadObjectDefinition(Tags tag, StreamLabel objectLabel
 
         internal Func<IFastSerializable> GetFactory(string fullName)
```

Comment:
@GrabYourPitchforks, you will be most interested in the changes to `GetFactory`.